pages/dashboard.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```

Comment:
UX Suggestion: On hover over the close button changing the cursor to a pointer would be preferable as per UX concerns.
![image](https://user-images.githubusercontent.com/91944618/154863535-e57a273c-8cb5-4a49-8f33-6ee6cfc3fee7.png)

Reply:
I'll do It later, I think It's a minor change so skipping it for now. Thanks!